tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Service/Microsoft.Legal.MatterCenter.Repository/SharePointInfo/Search.cs

```diff
@@ -133,7 +133,7 @@ public SearchResponseVM GetMatters(SearchRequestVM searchRequestVM)
                     var managedColumns = configuration.GetSection("ContentTypes").GetSection("ManagedColumns").GetChildren();
                     foreach (var key in managedColumns)
                     {
-                        managedProperties.Add("MC" + key.Value);
```

Comment:
Instead of hard coding "MC", moved this value to search settings in appsettings.json so that this value can be configurable